database/migration/src/main/resources/changesets/custom_label_entry.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet id="custom_LABEL_entry" author="rmunshi">
+        <createTable tableName="CUSTOM_LABEL_ENTRY">
+            <column autoIncrement="true" name="CUSTOM_LABEL_ENTRY_ID" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="PK_CUSTOM_LABEL_ENTRY"/>
+            </column>
+            <column name="CUSTOM_LABEL_KEY" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CUSTOM_LABEL_VALUE" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_UUID" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="rmunshi" id="WMSE_workflow_execution_uuid_foreign_key">
+        <addForeignKeyConstraint constraintName="FK_CUSTOM_LABEL_ENTRY_WORKFLOW_EXECUTION_UUID"
+                                 baseTableName="CUSTOM_LABEL_ENTRY"
+                                 baseColumnNames="WORKFLOW_EXECUTION_UUID"
+                                 referencedTableName="WORKFLOW_METADATA_SUMMARY_ENTRY"
+                                 referencedColumnNames="WORKFLOW_EXECUTION_UUID"
+                                 onDelete="CASCADE"
```

Comment:
Why is the `onDelete` here?  AFAIK nothing should ever be deleted from the referenced summary table.